gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/versions/RegionVersionVector.java

```diff
@@ -669,6 +669,11 @@ public void initializeVersionHolder(T mbr, RegionVersionHolder<T> otherHolder) {
         this.localExceptions.initializeFrom(vh);
       }
     } else {
+      if (mbr.equals(this.myId)) {
+        RegionVersionHolder<T> vh = otherHolder;
+        long version = vh.version;
+        updateLocalVersion(version);
+      }
```

Comment:
Why doesn't this need "this.localExceptions.initializeFrom(vh)" like in the code block above? Better will be to pull up the common code block at the start:

```java
if (mbr.equals(this.myId)) {
  RegionVersionHolder<T> vh = otherHolder;
  ...
} else if (h == null) {
  h = otherHolder.clone();
  ...
} else {
  ...
}
```